src/app/shared/stache/modules/video/video.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-video">
+  <iframe class="stache-video-embed" [src]="src" frameborder="0" allowfullscreen></iframe>
```

Comment:
I _do_ like having a `<stache-video>` component; just thinking about future implementations. If someone wanted an `iframe`, we'd have to create another component that looks very similar to this one. We can come back to this discussion in the future; just some thoughts.